grails-app/services/au/org/ala/bie/SearchService.groovy

```diff
@@ -883,16 +879,6 @@ class SearchService {
                             status: commonName.status,
                             priority: commonName.priority,
                             language: commonName.language ?: grailsApplication.config.commonNameDefaultLanguage,
-                            temporal: commonName.temporal,
```

Comment:
Part of the IEK work. Common names can now have details about what the name applies to,